nautobot/dcim/api/serializers.py

```diff
@@ -983,9 +982,9 @@ class PowerFeedSerializer(
     PathEndpointModelSerializerMixin,
     NautobotModelSerializer,
 ):
-    type = ChoiceField(choices=PowerFeedTypeChoices, default=PowerFeedTypeChoices.TYPE_PRIMARY)
-    supply = ChoiceField(choices=PowerFeedSupplyChoices, default=PowerFeedSupplyChoices.SUPPLY_AC)
-    phase = ChoiceField(choices=PowerFeedPhaseChoices, default=PowerFeedPhaseChoices.PHASE_SINGLE)
+    type = ChoiceField(choices=PowerFeedTypeChoices, required=False)
+    supply = ChoiceField(choices=PowerFeedSupplyChoices, required=False)
+    phase = ChoiceField(choices=PowerFeedPhaseChoices, required=False)
```

Comment:
What's the functional impact of these changes on the API behavior? Do they still default to the same/correct value if omitted entirely?

Should we add a blanket rule (maybe in pylint-nautobot) against ever specifying a `default` on any serializer ChoiceField or are there cases where it's valid/desirable?

Reply:
I think that setting field defaults should defer to the model field definition. There was a special case for TeamSerializer because the fields were part of a UniqueTogether so DRF automatically marked them as required.